src/steps/compute/index.ts

```diff
@@ -2191,22 +2215,31 @@ export const computeSteps: IntegrationStep<IntegrationConfig>[] = [
       },
     ],
     relationships: [
-      {
-        _class: RelationshipClass.USES,
-        _type: RELATIONSHIP_TYPE_IMAGE_USES_KMS_KEY,
-        sourceType: ENTITY_TYPE_COMPUTE_IMAGE,
-        targetType: ENTITY_TYPE_KMS_KEY,
-      },
       {
         _class: RelationshipClass.CREATED,
         _type: RELATIONSHIP_TYPE_SNAPSHOT_CREATED_IMAGE,
         sourceType: ENTITY_TYPE_COMPUTE_SNAPSHOT,
         targetType: ENTITY_TYPE_COMPUTE_IMAGE,
       },
     ],
-    dependsOn: [STEP_CLOUD_KMS_KEYS],
+    dependsOn: [],
```

Comment:
I notice that this step seems to depend on the `COMPUTE_SNAPSHOT` step. I would wager to bet that we're not producing the `SNAPSHOT_CREATED_IMAGE` relationships because of this missing dependency.

It'll be best to fully decouple these two steps, but since the _immediate_ need is to decouple steps from _different_ services, it's OK to keep this dependency for now.

```suggestion
    dependsOn: [STEP_COMPUTE_SNAPSHOTS],
```

from `getStepStartStates.ts': 

```ts
    [STEP_COMPUTE_IMAGES]: createStepStartState(ServiceUsageName.COMPUTE),
    ...
    [STEP_COMPUTE_SNAPSHOTS]: createStepStartState(ServiceUsageName.COMPUTE),
    ...
    [STEP_CLOUD_KMS_KEYS]: createStepStartState(ServiceUsageName.KMS),


```